docker/auto-setup.sh

```diff
@@ -140,12 +141,16 @@ setup_cassandra_schema() {
     { export CASSANDRA_PASSWORD=${CASSANDRA_PASSWORD}; } 2> /dev/null
 
     SCHEMA_DIR=${TEMPORAL_HOME}/schema/cassandra/temporal/versioned
-    temporal-cassandra-tool --ep "${CASSANDRA_SEEDS}" create -k "${KEYSPACE}" --rf "${CASSANDRA_REPLICATION_FACTOR}"
+    if [ "${SKIP_DB_CREATE}" != "true" ]; then
```

Comment:
I'd write this like
```
if [[ $SKIP_DB_CREATE != true ]]; then
```
since this is explicitly bash, doesn't have to be portable (bash doesn't require quoting inside `[[]]` in most cases)

Reply:
I will address this in separate PR.